resources/usb3_control/usb3_control.service

```diff
@@ -1,5 +1,5 @@
 [Unit]
-Description=USB hub control
+Description=USB 3 control
```

Comment:
We might need to consult this, what happened with USB hub service?

Reply:
I think it was just a fix of a typo. Look here, this is the name for **USB3 control**: https://github.com/airvolute/dcs-deploy/blob/custom_rootfs_and_batch/resources/usb3_control/usb3_control.service#L2

And this one is for **USB HUB control**: https://github.com/airvolute/dcs-deploy/blob/custom_rootfs_and_batch/resources/usb_hub_control/usb_hub_control.service#L2

So the problem was, that both USB 3 **AND** USB hub was named USB hub.

Reply:
Yep, it was a typo.